fundamentals/fundamentals-collections/src/main/scala/izumi/functional/IzEitherAggregations.scala

```diff
@@ -0,0 +1,103 @@
+package izumi.functional
+
+import izumi.functional.IzEitherAggregations.*
+import izumi.fundamentals.collections.nonempty.NonEmptyList
+
+import scala.collection.compat.*
+import scala.language.implicitConversions
+
+trait IzEitherAggregations {
+  @inline implicit final def EitherBiAggregate[L, R, Col[x] <: IterableOnce[x]](
+    col: Col[Either[NonEmptyList[L], R]]
+  ): EitherBiAggregate[L, R, Col] = new EitherBiAggregate(col)
+  @inline implicit final def EitherBiFlatAggregate[L, R, Col[x] <: IterableOnce[x], Col2[x] <: IterableOnce[x]](
+    col: Col[Either[NonEmptyList[L], Col2[R]]]
+  ): EitherBiFlatAggregate[L, R, Col, Col2] = new EitherBiFlatAggregate(col)
+  @inline implicit final def EitherBiFind[Col[x] <: IterableOnce[x], T](col: Col[T]): EitherBiFind[Col, T] = new EitherBiFind(col)
+  @inline implicit final def EitherBiFoldLeft[Col[x] <: IterableOnce[x], T](col: Col[T]): EitherBiFoldLeft[Col, T] = new EitherBiFoldLeft(col)
+  @inline implicit final def EitherBiMapAggregate[Col[x] <: IterableOnce[x], T](col: Col[T]): EitherBiMapAggregate[Col, T] = new EitherBiMapAggregate(col)
+  @inline implicit final def EitherBiFlatMapAggregate[Col[x] <: IterableOnce[x], T](col: Col[T]): EitherBiFlatMapAggregate[Col, T] = new EitherBiFlatMapAggregate(col)
+}
+
+object IzEitherAggregations extends IzEitherAggregations {
+
+  final class EitherBiAggregate[L, R, Col[x] <: IterableOnce[x]](private val col: Col[Either[NonEmptyList[L], R]]) extends AnyVal {
+    /** `sequence` with error accumulation */
+    def biAggregate(implicit b: Factory[R, Col[R]]): Either[NonEmptyList[L], Col[R]] = {
+      IzEither.EitherBiAggregate(col.iterator.map(_.left.map(_.toList)).toList)
```

Comment:
@YaroslavSahach This copies the entire collection to change from NonEmptyList to List to satisfy `Src[x] <: IterableOnce[x]` constraint on EitherBiAggregate. That would be inefficient. What you could do instead, is replace the constraint in BiAggregate with an implicit `Src[L] => IterableOnce[L]`:

```scala
  final class EitherBiAggregate[L, R, Src[_], Col[x] <: IterableOnce[x]](private val col: Col[Either[Src[L], R]]) extends AnyVal {
    /** `sequence` with error accumulation */
    def biAggregate(implicit b: Factory[R, Col[R]], ev: Src[L] => IterableOnce[L]): Either[List[L], Col[R]] = {
```

With the constraint relaxed that way, now you could implement this method without copying the entire list:

```suggestion
      new IzEither.EitherBiAggregate(col)
        .biAggregate(b, _.toList)
        .left.map(NonEmptyList.unsafeFrom)
```

Note that `NonEmptyList#toList` is an accessor method and will not rebuild the entire linked list.